Mobile/starter_project_dartsmiths/lib/features/example/presentation/widgets/article_reading.dart

```diff
@@ -0,0 +1,127 @@
+                                width: 15,
+                              ),
+                              Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: const [
+                                  Text(
+                                    'Richard Gervan',
+                                    style: TextStyle(color: Colors.blue),
+                                  ),
+                                  Text(
+                                    '2m ago',
+                                    style: TextStyle(color: Colors.blue),
+                                  )
+                                ],
+                              ),
+                            ],
+                          ),
+                          const Icon(Icons.bookmark_add_outlined)
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(20),
+                  ),
+                  height: 190,
+                  width: double.infinity,
+                  child: ClipRRect(
+                    borderRadius: const BorderRadius.only(
+                        topLeft: Radius.circular(20),
+                        topRight: Radius.circular(20)),
+                    child: Image.asset(
+                      'images/fashin3.jpg',
+                      width: double.infinity,
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+                const Padding(
+                    padding: EdgeInsets.fromLTRB(30, 25, 30, 0),
+                    child: Text(
+                        "That marked a turnaround from last year, when the social network reported a decline in users for the first time.\n The drop wiped billions from the firms market value.\n Since executives disclosed the fall in February, the firms share price has nearly halved.But shares jumped 19% in after-hours trade on Wednesday.",
+                        style: TextStyle(
+                          fontSize: 18,
+                          color: Colors.blueGrey,
+                          height: 1.5, // the height between text, default is null
+                          // letterSpacing: 1.0
+                        ))),
+              ],
+            ),
+          ),
+          Positioned(
+            bottom: 10,
+            right: 10,
+            child: TextButton(onPressed: (){},
+            style:
+              TextButton.styleFrom(backgroundColor: Colors.blue, padding: EdgeInsets.all(10)), 
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: const [
+                Icon(Icons.thumb_up_alt_outlined, color: Colors.white,),
+                SizedBox(width: 10,),

```

Comment:
Remove all the static sizing data. There are others in the code this is an example. Use media query

Reply:
Done